test/spec/features/search-pad/SearchPadSpec.js

```diff
@@ -139,6 +139,17 @@ describe('features/searchPad', function() {
           } ];
         }
 
+        if (pattern === 'modern') {
```

Comment:
I think we should check _legacy_ and treat _modern_ as just the new default. But maybe I miss something fundamental here

Reply:
Related to https://github.com/bpmn-io/diagram-js/pull/932#discussion_r1827399644. 

`modern` is just a name here (for testing), and internally. Both approaches are just fine. We could of course decide to do a full-fledged migration, major breaking change.